src/renderer/Components/Home/index.jsx

```diff
@@ -3,15 +3,21 @@ import { useEffect } from "react";
 import { signInWithCustomToken } from "firebase/auth";
 import { useNavigate, useSearchParams } from "react-router-dom";
 
-import { auth } from "@renderer/firebase";
+import { auth } from "../../../../firebase";
 import usePackageStore from "@renderer/store";
 
+import { GUIDE_MESSAGES } from "../../constants/messages";
```

Comment:
```suggestion
import { GUIDE_MESSAGES } from "@renderer/constants/messages";
```
여기는 @renderer 절대경로 사용할 수 있습니다 :)

Reply:
절대 경로를 사용하니 간결해져서 좋네요!